src/data/actions/grades.js

```diff
@@ -39,6 +40,18 @@ const gotGrades = (grades, cohort, track, assignmentType, headings, prev, next,
   courseId,
 });
 
+const gotGradeOverrideHistory = (
+  overrideHistory, currentEarnedAllOverride, currentPossibleAllOverride,
+  currentEarnedGradedOverride, currentPossibleGradedOverride,
```

Comment:
I would recommend using an object parameter to get named arguments, rather than relying on positional arguments. My usual threshold for when to switch from positional to named is about 3 arguments.

More concrete: https://2ality.com/2011/11/keyword-parameters.html

Reply:
I understand the idea behind it, but would it be worth doing it in this case, since the function is called with another object's properties that also have descriptive names.

Reply:
This is absolutely worth doing. It's an extremely easy refactor.
```javascript
const gotGradeOverrideHistory = ({
  overrideHistory, currentEarnedAllOverride, currentPossibleAllOverride,
  currentEarnedGradedOverride, currentPossibleGradedOverride,
  }) => ({
```
and at call sites you just add those same parameter names:
```javascript
        dispatch(gotGradeOverrideHistory({
          overrideHistory: formatGradeOverrideForDisplay(data.history),
          ...
        })
```

Reply:
Thank you. Will do.